Backpack/src/androidTest/java/net/skyscanner/backpack/text/internal/FontFamilyResolverTest.kt

```diff
@@ -0,0 +1,61 @@
+package net.skyscanner.backpack.text.internal
+
+import android.content.Context
+import android.graphics.Typeface
+import android.view.ContextThemeWrapper
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import net.skyscanner.backpack.test.R
+import net.skyscanner.backpack.text.BpkText
+import net.skyscanner.backpack.text.FontCache
+import net.skyscanner.backpack.util.BpkTheme
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class FontFamilyResolverTest {
+  private lateinit var context: Context
+
+  @Before
+  fun setUp() {
+    context = InstrumentationRegistry.getInstrumentation().targetContext
+  }
+
+  @Test
+  fun test_default_font() {
+    val subject = FontFamilyResolver(BpkTheme.wrapContextWithDefaults(context))
+
+    val serif = Typeface.create("sans-serif", Typeface.NORMAL)
+    val serifMedium = Typeface.create("sans-serif-medium", Typeface.NORMAL)
+    val serifBlack = Typeface.create("sans-serif-black", Typeface.NORMAL)
+
+    Assert.assertEquals(serif, subject.getForWeight(BpkText.Weight.NORMAL))
```

Comment:
I'd spilled it into several tests.

Reply:
I'll just add a message to the assert as this is really only testing one function, this will make it clear which on is which if the test fails